.gitignore

```diff
@@ -13,3 +13,5 @@ examples/frameworks/pylonstest/PasteScript*
 examples/frameworks/pylonstest/pylonstest.egg-info/
 examples/frameworks/django/testing/testdb.sql
 .tox
+venv
```

Comment:
Added the venv because i figured the Makefile made assumptions about the location of the venv but I've removed that now.